### PR TITLE
chore(submodules):Using relative URL for submodule

### PR DESCRIPTION
**Pull request by jvictorcf**
### Change description ###
The idea here is to make it easier to make changes to the submodule easier by using the same URL pattern used for the parent project.
This way, pushing changes will work whether you've used ssh or https to clone the parent project.

For example, I use ssh and therefore don't want to type my password to push changes to fpla-docker.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "fpla-docker"]
 	path = fpla-docker
-	url = https://github.com/hmcts/fpla-docker
+	url = ../fpla-docker.git
```
Comment (jvictorcf) [inline .gitmodules:3]: I'd be happy to use "../../hmcts/fpla-docker.git" (for clarity) instead, if the team prefers.

Comment (lewisbirks) [reply .gitmodules:3]: I think it doesn't hurt to have a bit of clarity with this, so do submodule urls work the same way as file paths?

Comment (tomaszpowroznik) [reply .gitmodules:3]: if build is green then we are ok, jenkins pipeline rely on submodule and scripts there

**Review by tomaszpowroznik — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (jvictorcf) [reply .gitmodules:3]: Cool. I change it.

**Review by tomaszpowroznik — APPROVED** [review]

**Pull request merged**